addons/digest/wizard/digest_test_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
Technically the file should be ``digest_digest_test_views.xml`` (or alternatively rename the wizard model to ``digest.test``).

Reply:
Opted for changing the model name to digest.test as there already exists digest.tip that follow the same pattern.